doc/builders/images/binarycache.section.md

```diff
@@ -0,0 +1,13 @@
+# pkgs.mkBinaryCache {#sec-pkgs-binary-cache}
+
+`pkgs.mkBinaryCache` is a function for creating Nix flat-file binary caches. Such a cache exists as a directory on disk, and can be used as a Nix substituter by passing `--substituter file:///path/to/cache` to Nix commands.
+
+The parameters of `mkBinaryCache` with an example value are described below. This derivation will construct a flat-file binary cache containing the closure of `hello`.
+
+```nix
+mkBinaryCache {
+  rootPaths = [hello];
+}
+```
+
+- `rootPaths` specifies a list of root derivations. The transitive closure of these derivations will be copied into the cache.
```

Comment:
Use a definition list, they better match the semantics of what we want to achieve here.

```suggestion
`rootPaths` _(list of derivations)_

: List of root derivations. The transitive closure of these derivations will be copied into the cache.
```

Reply:
@fricklerhandwerk FWIW, no other documentation in the `images/` folder does this. They all use normal lists.

Reply:
Yeah true, we don't have any standard yet and I think it would be worth writing it down. (I would if time was not severely limited and the list of "would be great to have" items was not growing without bounds.) 

Like all reviews here, this one about definition lists is just my personal opinion and I can't and don't want to tell anyone what to do. But I think more precise semantics is always better, and the cost here is negligible (in fact I think it's already a short term saving, because it already has more structure in the markdown source than unordered lists) and I try to keep it down for you offering a ready-made diff to pick up.

I'd rather have someone stumble over inconsistent formatting and possibly offer a fix than perpetuating a strictly inferior approach. Feel free to disagree. :)

Reply:
I would say "strictly inferior" is strong words when talking about a weird extended Markdown syntax that most people don't remember :P. And also implementing such changes in a scattershot way and hoping someone comes along later to fix the rest is the wrong way to do things; it should be in its own PR. I'd prefer not to be the person who messes up the consistency of the docs in this PR.

Reply:
Markdown just happens to be the agreed-upon least terrible tool we have available. I fully understand your sentiment about introducing inconsistency. Do you have a proposal that has a high chance to actually be implemented? My intuition is, and I may well be wrong, that making changes incrementally while people are already at it, and discussing the merits, as we do now, even at the cost of temporary and potentially long-standing inconsistencies is much more probable to build up momentum for substantial change. Big Bang updates are risky and require much more dedication to pull through. Maybe I'm overestimating the total effort to convert all argument lists to a different format, for example, but there are many sweeping changes, not just in documentation, that many will welcome and still no one has done.

Anyway, just my two cents.

Reply:
I think I'll switch this PR back to a normal list if that's all right with you. I consider it bad form to introduce inconsistency. In the current world, switching to definition lists is an unrelated change and unrelated changes should not be piggybacked onto random PRs. I agree with you definition lists look slightly better when formatted, and I'd encourage you to write a PR to switch the docs over! Doing it in one fell swoop is infinitely preferable IMO because then there's a clear/unified place for discussion to happen, and a clear single commit to blame/revert if needed.